xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -117,6 +117,7 @@ SelectionStream& CSelectionStreams::Get(StreamType type, int index)
 
 std::vector<SelectionStream> CSelectionStreams::Get(StreamType type)
 {
+  CSingleLock lock(m_section);
```

Comment:
Please tell me where class CSelectionStreams is accessed from outside of the player thread. I found nothing. Or more precise: it is used in the CVideoPlayer constructor, at a time where the player thread has not yet been spawned. That is safe.

Am I right to assume that locking in CSelectionStreams is completely unnecessary?